src/main/java/me/aborozdykh/amazonreview/entity/Review.java

```diff
@@ -0,0 +1,104 @@
+package me.aborozdykh.amazonreview.entity;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * @author Andrii Borozdykh
+ */
+@Entity
+@Table(name = "review")
+public class Review {
+    @Id
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "product_id")
+    private Product product;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+    private int helpfulnessNumerator;
```

Comment:
What the property it is?

Reply:
Could you please explain what do you mean?

Reply:
resolved